src/JsonApi/Request/JsonApiRequestBuilder.php

```diff
@@ -371,7 +350,8 @@ private function getMediaTypeWithProfiles(array $profiles, bool $compatibility):
         $result = "application/vnd.api+json";
 
         if (empty($profiles) === false) {
-            $result .= ';profile="' . implode(" ", $profiles) . '"' . ($compatibility ? ',application/vnd.api+json' : "");
+            $result .= ';profile="' . implode(" ", $profiles) . '"' . ($compatibility ? ',application/vnd.api+json'
+                    : "");
```

Comment:
Is this an intentional change?

Reply:
It has been changed after apply csfixer

Reply:
But looks a little bit weird. I've fix it